ci/docker-publish.yml

```diff
@@ -41,9 +41,9 @@ jobs:
       # https://github.com/sigstore/cosign-installer
       - name: Install cosign
         if: github.event_name != 'pull_request'
-        uses: sigstore/cosign-installer@1e95c1de343b5b0c23352d6417ee3e48d5bcd422
+        uses: sigstore/cosign-installer@dbb3160ad8a99990fa646eb85bfdebcb52ecfeed
```

Comment:
How did you determine this commit?

Release v1.4.1 is a different commit: https://github.com/sigstore/cosign-installer/releases/tag/v1.4.1
https://github.com/sigstore/cosign-installer/commit/116dc6872c0a067bcb78758f18955414cdbf918f